src/main.js

```diff
@@ -17,3 +15,51 @@ var currentCover;
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
+
+
+var currentCover = document.querySelector('.cover-image');
+currentCover.src = covers[getRandomIndex(covers)];
+var currentTitle = document.querySelector('.cover-title');
+currentTitle.innerText = titles[getRandomIndex(titles)];
+var firstDescriptor = document.querySelector('.tagline-1');
+firstDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
+var secondDescriptor = document.querySelector('.tagline-2');
+secondDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
+
```

Comment:
I think that this was a great way to pass Iteration 0, but I'm excited to see if we can condense it Iteration 1.